src/v2/view-builder/ViewFactory.js

```diff
@@ -36,6 +36,9 @@ const VIEWS_MAPPING = {
   'device-apple-sso-extension': {
     [DEFAULT]: SSOExtensionView,
   },
+  'cancel-transaction': {
+    [DEFAULT]: SSOExtensionView,
```

Comment:
can we do something better here, where different keys have same values ?? Can we reuse existing ?

Reply:
for instance `device-apple-sso-extension` and `cancel-transaction` use same View.

Reply:
@pradeepdewda-okta the API design has been reviewed and remediations are named differently on purpose. They are both for SSO Extension approach in different step of the flow. They do have the same appearance and need the same trigger from the view level thus share the same view. But what they actually do is different, the idx actions may have different method/data.

Reply:
I was trying to suggest what the code is doing at line 89 in this file to get the step view. If it's not feasible then it's fine.